pkg/action/reset_test.go

```diff
@@ -108,22 +107,24 @@ var _ = Describe("Reset action tests", func() {
 					},
 					{
 						Name:            "device2",
-						FilesystemLabel: "COS_STATE",
+						FilesystemLabel: "COS_OEM",
 						FS:              "ext4",
+						MountPoint:      "/oem",
 					},
 					{
 						Name:            "device3",
-						FilesystemLabel: "COS_PERSISTENT",
+						FilesystemLabel: "COS_RECOVERY",
 						FS:              "ext4",
+						MountPoint:      "/run/initramfs/cos-state",
```

Comment:
@Itxaka wait, this one says recovery, but mountpoint cos-state? is this actually how it's supposed to be? if so, can you clarify?

Reply:
Historical reasons

*Refuses to elaborate*

Reply:
Basically to keep the ongoing theme, when booting on recovery, the recovery partition is mounted as it was the state partition in the same path as the state partition is mounted during active/passive

No other reason than that